Makefile

```diff
@@ -17,7 +17,7 @@ dev: verify test ## Run all steps in the developer loop
 
 ci: verify licenses battletest ## Run all steps used by continuous integration
 
-release: publish helm docs ## Run all steps in release workflow
+release: codegen publish helm docs ## Run all steps in release workflow
```

Comment:
What changed to necessitate including this here?

Reply:
I realized it was missing. It's just an extra safe check that all of our files have the correct headers, that apidocs are up to date, etc.